_translator-files/po/sv-SE/Directories.po

```diff
@@ -1,112 +1,113 @@
 # SOME DESCRIPTIVE TITLE
 # Copyright (C) YEAR Free Software Foundation, Inc.
 # This file is distributed under the same license as the PACKAGE package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
-#, fuzzy
+# Daniel <daniel@mailsystem.net>, 2024.
+# ignotus <ignotus666@users.noreply.hosted.weblate.org>, 2024.
 msgid ""
 msgstr ""
 "Project-Id-Version: PACKAGE VERSION\n"
-"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
-"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
+"PO-Revision-Date: 2024-08-05 13:45+0000\n"
+"Last-Translator: Daniel <daniel@mailsystem.net>\n"
 "Language-Team: LANGUAGE <LL@li.org>\n"
-"Language: \n"
+"Language: sv-SE\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+"X-Generator: Weblate 5.7-dev\n"
 
 #. type: Yaml Front Matter Hash Value: lang
 #: ../wiki/en/Directories.md:1
 #, no-wrap
 msgid "en"
-msgstr ""
+msgstr "sv-SE"
 
 #. type: Yaml Front Matter Hash Value: layout
 #: ../wiki/en/Directories.md:1
 #, no-wrap
 msgid "wiki"
-msgstr ""
+msgstr "wiki"
 
 #. type: Yaml Front Matter Hash Value: permalink
 #: ../wiki/en/Directories.md:1
 #, no-wrap
 msgid "/wiki/Directories"
-msgstr ""
+msgstr "/wiki/Directories"
 
 #. type: Yaml Front Matter Hash Value: title
 #: ../wiki/en/Directories.md:1
 #, no-wrap
 msgid "Directories"
-msgstr ""
+msgstr "Kataloger"
 
 #. type: Plain text
 #: ../wiki/en/Directories.md:9
 #, markdown-text
 msgid "{% include breadcrumb.html root=\"More\" branch1=\"Server Administration\" branch1-url=\"Running-a-Server\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Mer\" branch1=\"Serveradministration\" branch1-url=\"Köra-en-server\" %}"
```

Comment:
" branch1-url=\"Running-a-Server\" shouldn't be translated.

Reply:
This one is crucial to get right.